reports/crash_stats_oom/getting-crash-stats-for-OOM-data-to-S3.ipynb

```diff
@@ -152,12 +152,12 @@
     "        'Content-Type': 'application/json; charset=UTF-8'\n",
     "    }\n",
     "    target = 'https://crash-stats.mozilla.com/api/SuperSearch/'\n",
-    "    per_page_default = 100\n",
+    "    per_page_default = 500\n",
     "\n",
     "    param_data = {}\n",
     "    param_data['release_channel'] = \"beta\"\n",
     "    param_data['process_type'] = \"content\"\n",
-    "    param_data['_facets'] = \"signature\"\n",
+    "    # param_data['_facets'] = \"signature\"\n",
```

Comment:
Please remove this commented-out line (or add a comment explaining why it's there if you want to keep it).